playbooks/openshift-node/private/bootstrap.yml

```diff
@@ -8,6 +8,8 @@
     set_stats:
       data:
         installer_phase_node:
+          title: "Node Preparation"
+          playbook: "(no entry point playbook)"
```

Comment:
Lets make `playbook` optional and show `This phase can be restarted` only if its set?